src/reducers/versions.tsx

```diff
@@ -161,6 +161,7 @@ type InternalVersionFile = {
   created: string;
   downloadURL: string | null;
   id: number;
+  sha256: string | null;
   size: number;
 };
 
```

Comment:
I believe the `VersionFile` should also be updated.

Reply:
`VersionFile` already had property `sha256: string`, which gets the value from `entry.sha256` before this patch. What update do you mean?

Reply:
But that value could be `null`, no?

Reply:
`getVersionFile` deals with the case `sha256===null`: outputs debug info and returns `undefined`, so the returned `VersionFile` always has string `sha256`.

Reply:
ha, oops. Thanks for the explanation!